net/pdns/Makefile

```diff
@@ -0,0 +1,203 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=pdns
+PKG_VERSION:=4.2.0-rc1
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
+PKG_SOURCE_URL:=https://downloads.powerdns.com/releases/
+PKG_HASH:=806362a6332139267c4fe888c717a35a5658438c894c6764f12be7184367e61e
+
+PKG_MAINTAINER:=James Taylor <james@jtaylor.id.au>
+PKG_LICENCE:=GPL-2.0-only
+PKG_LICENCE_FILES:=COPYING
+
+PKG_INSTALL:=1
+
+PKG_BUILD_DEPENDS:=unixodbc/host
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/pdns/Default
+  SECTION:=net
+  CATEGORY:=Network
+  SUBMENU:=IP Addresses and Names
+  TITLE:=PowerDNS Authoritative Server
+  DEPENDS:=+libatomic +libopenssl +libsqlite3 +protobuf +libsodium +boost +boost-program_options +libcurl +p11-kit +lua +liblua
```

Comment:
libopenssl, liblua, and libatomic can be removed. They are implied by other dependencies.

Reply:
I'm refactoring the dependencies into each package based on https://repo.powerdns.com/debian/dists/stretch-auth-42/main/binary-amd64/Packages